lib/logstash/inputs/gelf.rb

```diff
@@ -29,9 +30,12 @@ class LogStash::Inputs::Gelf < LogStash::Inputs::Base
   # The IP address or hostname to listen on.
   config :host, :validate => :string, :default => "0.0.0.0"
 
-  # The port to listen on. Remember that ports less than 1024 (privileged
+  # The ports to listen on. Remember that ports less than 1024 (privileged
   # ports) may require root to use.
+  # port_tcp and port_udp can be used to have a different port for udp than the tcp port.
   config :port, :validate => :number, :default => 12201
+  config :port_tcp, :validate => :number, :default => 0
```

Comment:
I suggest removing the `:default => 0`, if the user doesn't set it, the value is nil

Reply:
Done